patientsearch/config.py

```diff
@@ -14,11 +14,11 @@ def load_json_config(potential_json_string):
         return json.loads(potential_json_string)
 
 
-
 LANDING_INTRO = os.getenv("LANDING_INTRO", "")
 LANDING_BUTTON_TEXT = os.getenv("LANDING_BUTTON_TEXT", "")
 LANDING_BODY = os.getenv("LANDING_BODY", "")
 
+MORE_MENU = os.getenv("MORE_MENU", [])
```

Comment:
@pbugni this seems to be reading the default empty array as literally a string .i.e. `"[]"` (see attached):

![Screen Shot 2021-11-02 at 1 48 22 PM](https://user-images.githubusercontent.com/12942714/139949501-8100b732-7c92-4552-8d29-0cffca017bec.png)